### PR TITLE
{bio} [intel/2016b] kaiju/1.5.0 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic66 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/14c24b572e0d6931f9808a371eb9a005 for a full test report.

File: easybuild/easyconfigs/k/Kaiju/Kaiju-1.5.0-makefile.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff -ur kaiju-1.5.0.orig/src/bwt/Makefile kaiju-1.5.0/src/bwt/Makefile
```
Comment (boegel) [inline easybuild/easyconfigs/k/Kaiju/Kaiju-1.5.0-makefile.patch]: @wpoely86 patch description + author?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2127.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b90e19dcd67b78eaca7e0698292103ed for a full test report.

**Comment by wpoely86** [comment]
@boegel done

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic66 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/92e3b4b6f26f1e067a2b9f988817742e for a full test report.

**Comment by wpoely86** [comment]
unbroken it!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2522.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1cb52c65cd119aded9072010bbc5670b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**